cmd/kube-proxy/app/server.go

```diff
@@ -736,7 +736,7 @@ func (s *ProxyServer) Run() error {
 	serviceConfig.RegisterEventHandler(s.Proxier)
 	go serviceConfig.Run(wait.NeverStop)
 
-	if s.UseEndpointSlices {
+	if s.UseEndpointSlices && s.ProxyMode != proxyModeUserspace {
```

Comment:
I think this logic belongs in `server_others.go` (and `server_windows.go`); it should not set `UseEndpointSlices` if `ProxyMode` is a mode that doesn't support it.

Reply:
ohh, sorry, updated.